pkg/helper/cardinality/cardinality.go

```diff
@@ -0,0 +1,85 @@
+package cardinality
+
+import (
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"strings"
+)
+
+// The cardinalities relates to how the field is suitable for usage as a metric label wrt cardinality; it may have 3 values: fine, careful, avoid
+type Warn string
+
+const (
+	WarnAvoid   Warn = "avoid"
+	WarnCareful Warn = "careful"
+	WarnFine    Warn = "fine"
+	WarnUnknown Warn = "unknown"
+)
+
+//go:embed cardinality.json
+var rawCardinality []byte
+var cardinality *map[string]Warn
```

Comment:
`map` is already a reference we don't need a pointer here

Reply:
done in https://github.com/netobserv/network-observability-operator/pull/809/commits/5eecbd5f758b2da4a2b422b57567de0a5fa0a8af